packages/blockchain-wallet-v4-frontend/src/layouts/Wallet/MenuLeft/Navigation/template.js

```diff
@@ -270,7 +278,7 @@ const Navigation = props => {
             target='_blank'
             style={{ width: '100%' }}
           >
-            {renderPitSidenav()}
+            {renderPitSidenav(null, true)}
```

Comment:
can you just pass `props.isPitAccountLinked` instead of `true`?  it will be slightly more readable